package.json

```diff
@@ -12,6 +12,12 @@
       "name": "Elf Pavlik"
     }
   ],
+  "files": [
+    "src/**/*.js",
+    "index.js",
+    "types"
+  ],
+  "main": "index.js",
```

Comment:
I added this but notice that it's an empty file. 

@alien-mcl, I can remove here but ideally we should publish a default export in main file

Reply:
I don't understand - which is an empty file?

Reply:
https://github.com/HydraCG/Heracles.ts/blob/master/index.ts

0 bytes :)